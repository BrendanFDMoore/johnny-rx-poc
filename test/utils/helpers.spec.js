import * as helpers from 'utils/helpers';

describe('helpers/createRequestTypes', () => {
  it('should return a proper object', () => {
    expect(helpers.createRequestTypes('REQUEST')).toMatchSnapshot();
  });
});

describe('helpers/datasetToObject', () => {
  const el = document.createElement('div');
  el.setAttribute('data-rule', 'yes');
  el.setAttribute('data-minute-maid', 'no');

  it('should convert DOMElement data to object', () => {
    expect(helpers.datasetToObject(el)).toMatchSnapshot();
  });
});
