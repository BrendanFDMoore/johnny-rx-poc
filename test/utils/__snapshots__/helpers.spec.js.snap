exports[`helpers/createRequestTypes should return a proper object 1`] = `
Object {
  "FAILURE": "REQUEST_FAILURE",
  "REQUEST": "REQUEST_REQUEST",
  "SUCCESS": "REQUEST_SUCCESS",
}
`;

exports[`helpers/datasetToObject should convert DOMElement data to object 1`] = `
Object {
  "minuteMaid": "no",
  "rule": "yes",
}
`;
